### PR TITLE
fix: add missed labels to elements

### PR DESCRIPTION
**Pull request by stasiukanya**

### CODE REVIEW

**Review by miranda76 — APPROVED** [review]

**Review by miranda76 — APPROVED** [review]

File: demo/ComboBox.tsx [hunk 1]
```diff
@@ -211,6 +211,7 @@ export default class ComboBox extends React.Component<ComboBoxProps, ComboBoxSta
           onFocus={this.open}
           onBlur={this.handleBlur}
           onKeyDown={this.handleKeyPress}
+          aria-label="URL to a spec to try"
```
Comment (RomanHotsiy) [inline demo/ComboBox.tsx:214]:
```suggestion
          aria-label="URL to an OpenAPI definition to try"
```

File: demo/index.tsx [hunk 2]
```diff
@@ -79,7 +79,10 @@ class DemoApp extends React.Component<
       <>
         <Heading>
           <a href=".">
-            <Logo src="https://github.com/Redocly/redoc/raw/master/docs/images/redoc-logo.png" />
+            <Logo
+              src="https://github.com/Redocly/redoc/raw/master/docs/images/redoc-logo.png"
+              alt="Redoc"
```
Comment (RomanHotsiy) [inline demo/index.tsx:84]:
```suggestion
              alt="Redoc logo"
```

File: src/components/JsonViewer/JsonViewer.tsx [hunk 3]
```diff
@@ -51,7 +51,9 @@ class Json extends React.PureComponent<JsonProps> {
   expandAll = () => {
     const elements = this.node.getElementsByClassName('collapsible');
     for (const collapsed of Array.prototype.slice.call(elements)) {
-      (collapsed.parentNode as Element)!.classList.remove('collapsed');
+      const parentNode = collapsed.parentNode as Element;
+      parentNode!.classList.remove('collapsed');
```
Comment (RomanHotsiy) [inline src/components/JsonViewer/JsonViewer.tsx:55]:
```suggestion
      parentNode.classList.remove('collapsed');
```

File: src/components/JsonViewer/JsonViewer.tsx [hunk 4]
```diff
@@ -61,7 +63,9 @@ class Json extends React.PureComponent<JsonProps> {
     const elementsArr = Array.prototype.slice.call(elements, 1);
 
     for (const expanded of elementsArr) {
-      (expanded.parentNode as Element)!.classList.add('collapsed');
+      const parentNode = expanded.parentNode as Element;
+      parentNode!.classList.add('collapsed');
```
Comment (RomanHotsiy) [inline src/components/JsonViewer/JsonViewer.tsx:67]:
```suggestion
      parentNode.classList.add('collapsed');
```

**Review by RomanHotsiy — APPROVED** [review]
LGTM, plz check the suggested changes

**Pull request merged**